diff --git a/src/driver/amdxdna/amdxdna_pci_drv.c b/src/driver/amdxdna/amdxdna_pci_drv.c
index b4819e9..c629f08 100644
--- a/src/driver/amdxdna/amdxdna_pci_drv.c
+++ b/src/driver/amdxdna/amdxdna_pci_drv.c
@@ -88,7 +88,9 @@ static int amdxdna_probe(struct pci_dev *pdev, const struct pci_device_id *id)
 #if KERNEL_VERSION(6, 10, 0) > LINUX_VERSION_CODE
 	drmm_mutex_init(&xdna->ddev, &xdna->dev_lock);
 #else
-	devm_mutex_init(dev, &xdna->dev_lock);
+	ret = devm_mutex_init(dev, &xdna->dev_lock);
+  if (ret)
+    return ret;
 #endif
 	init_rwsem(&xdna->notifier_lock);
 	INIT_LIST_HEAD(&xdna->client_list);
