diff --git a/src/driver/amdxdna/aie2_ctx.c b/src/driver/amdxdna/aie2_ctx.c
index 51e3ac5..dc27f6f 100644
--- a/src/driver/amdxdna/aie2_ctx.c
+++ b/src/driver/amdxdna/aie2_ctx.c
@@ -967,8 +967,12 @@ int aie2_cmd_submit(struct amdxdna_ctx *ctx, struct amdxdna_sched_job *job,
 		ret = -ENOMEM;
 		goto rq_yield;
 	}
-
-	ret = drm_sched_job_init(&job->base, &ctx->priv->entity, 1, ctx);
+  
+  //struct drm_sched_rq *rq = ctx->priv->entity.rq;
+  //struct drm_sched_fence *fence;
+  //u32 credits = rq->base.weight;  // adjust if needed
+  //void *owner = &ctx->priv->entity;
+  ret = drm_sched_job_init(&job->base, &ctx->priv->entity, 1, &ctx->priv->entity, 0);
 	if (ret) {
 		XDNA_ERR(xdna, "DRM job init failed, ret %d", ret);
 		goto free_chain;
