diff --git a/src/runtime_src/core/tools/xbtracer/CMakeLists.txt b/src/runtime_src/core/tools/xbtracer/CMakeLists.txt
index 2276159fb..c2ded9742 100644
--- a/src/runtime_src/core/tools/xbtracer/CMakeLists.txt
+++ b/src/runtime_src/core/tools/xbtracer/CMakeLists.txt
@@ -26,6 +26,7 @@ include_directories (
 ${Protobuf_INCLUDE_DIR}
 ${CMAKE_CURRENT_BINARY_DIR}
 )
+find_package(absl REQUIRED)
 # Generate Cpp files from Proto file
 file(GLOB PROTO_SRC_FILES
   "${CMAKE_CURRENT_SOURCE_DIR}/src/*.proto"
@@ -63,7 +64,7 @@ add_library(xrt_trace SHARED ${XBTRACER_WRAPPER_SRC_FILES} ${ProtoHeaders})
 set_target_properties(xrt_trace PROPERTIES VERSION ${XRT_VERSION_STRING} SOVERSION ${XRT_SOVERSION})
 target_compile_definitions(xrt_trace PRIVATE XRT_ABI_VERSION=${XRT_VERSION_MAJOR})
 
-target_link_libraries(xrt_trace PRIVATE xbtracer_common xbtracer_protobuf ${Protobuf_LIBRARIES} xrt_coreutil)
+target_link_libraries(xrt_trace PRIVATE absl::log absl::log_internal_check_op xbtracer_common xbtracer_protobuf ${Protobuf_LIBRARIES} xrt_coreutil)
 add_dependencies(xrt_trace xbtracer_common xbtracer_protobuf xrt_coreutil)
 
 file(GLOB XBTRACER_CAPTURE_SRC_FILES
@@ -78,7 +79,7 @@ file(GLOB XBREPLAY_SRC_FILES
   "${CMAKE_CURRENT_SOURCE_DIR}/src/replay/*.cpp"
 )
 add_executable(xrt-replay ${XBREPLAY_SRC_FILES})
-target_link_libraries(xrt-replay PRIVATE xbtracer_common xbtracer_protobuf ${Protobuf_LIBRARIES} xrt_coreutil)
+target_link_libraries(xrt-replay PRIVATE absl::log_internal_check_op xbtracer_common xbtracer_protobuf ${Protobuf_LIBRARIES} xrt_coreutil)
 if (NOT WIN32)
   target_link_libraries(xrt-replay PRIVATE pthread)
 endif (NOT WIN32)

